frontend/src/components/common/player/common/constants.ts

```diff
@@ -0,0 +1,15 @@
+const ARRAY_SHIFT = 1;
+const FIRST_ITEM_IN_ARRAY = 0;
```

Comment:
```suggestion
const FIRST_ARRAY_IDX = 0;
```
move it to `shared/common/constants/array`